lib/releaselib.sh

```diff
@@ -709,7 +709,7 @@ release::gcs::publish () {
 
     # If public, validate public link
     logecho -n "* Validating uploaded version file at $public_link: "
-    contents="$(curl -Ls $public_link)"
+    contents="$(curl --retry=5 -Ls $public_link)"
```

Comment:
Might be worth pulling common curl options into a $CURL var in gitlib.sh and using it directly.  Not blocking however. 
/lgtm